readme.md

```diff
@@ -139,8 +166,14 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
-
+function Instructor(name){
+this.name = name;
+this.receivePresent = function(gift){
+console.log(name + " promptly drops the " + gift + " on the floor.");
+}}
+
+var andy = new Instructor("Andy");
+andy.receivePresent("Resin Laying Deer Figurine, Gold");
 ```
 
```

Comment:
:+1: Definitely works, but just a note that It may be more appropriate to write the `receivePresent` function as an instance method  i.e.
```
function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}
```